jfix-stdlib-batching/src/main/java/ru/fix/stdlib/batching/BatchingManagerThread.java

```diff
@@ -70,6 +67,8 @@ public BatchingManagerThread(
         this.profiler = profiler;
         this.batchManagerId = batchManagerId;
         this.batchTask = batchTask;
+        this.batchManagerThreadAwaitMetric = "BatchingManagerThread." + batchManagerId + ".batch_processor_thread_await";
```

Comment:
Looks like "batch_processor" part is redudant in "batch_processor_thread_await". Does  "batch_processor"  add new information?

Reply:
This means we are sitting in BatchManagerThread (with own single thread pool) and waiting for free thread for BatchProcessor form dynamic batch processor's pool